sql/2022/12/opportunity-score-wp-sites-lcp-without-fetchpriority-high.sql

```diff
@@ -0,0 +1,96 @@
+# HTTP Archive query to get Performance Lab plugin version distribution in a given month.
+#
+# WPP Research, Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See query results here: https://github.com/GoogleChromeLabs/wpp-research/pull/15
+SELECT 
+  lh._TABLE_SUFFIX AS `Client`,
+  COUNT(DISTINCT lh.url) AS `With_fetchpriority_on_lcp`, 
+  (total-COUNT(DISTINCT lh.url)) AS `Without_fetchpriority_on_LCP`,
+  total AS `Total_with_LCP`,
+  totalwp AS `Total_WP_sites`,
+  CONCAT(ROUND((total-COUNT(DISTINCT lh.url))*100/total, 3),' %') AS `Opportunity_Score`,
+  CONCAT(ROUND((total-COUNT(DISTINCT lh.url))*100/totalwp, 3),' %') AS `Overall_Opportunity_Score`
+FROM 
+  `httparchive.lighthouse.2022_10_01_*` AS lh
+JOIN
+  `httparchive.technologies.2022_10_01_*` AS tech
+ON
+  tech.url = lh.url
+JOIN
+(
+  SELECT 
+    lh._TABLE_SUFFIX, 
+    COUNT(DISTINCT lh.url) AS total 
+  FROM 
+    `httparchive.lighthouse.2022_10_01_*` AS lh 
+  JOIN 
+    `httparchive.technologies.2022_10_01_*` AS tech 
+  ON 
+    tech.url = lh.url 
+  WHERE 
+    lh._TABLE_SUFFIX = tech._TABLE_SUFFIX 
+  AND 
+    app = 'WordPress' 
+  AND 
+    category = 'CMS' 
```

Comment:
Is it necessary to pass category if we get a proper result by using the "app" field? 

Reply:
@mukeshpanchal27  there here 2 categories for WordPress for each URL. So, if we only pass app, it will return us two rows for a single URL in the results by default (unless you are using distinct). Explicit mentioning of category gets us only one row and thereby the entire query runs a bit faster.